lib/auth/webauthncli/fido2_test.go

```diff
@@ -1140,32 +1140,6 @@ func TestFIDO2Login_errors(t *testing.T) {
 			prompt:    prompt,
 			wantErr:   context.DeadlineExceeded.Error(),
 		},
-		{
-			name:      "nil origin",
```

Comment:
I'd keep "nil origin" and at least one assertion validation, so we can make sure the validations are "plugged" correctly.